### PR TITLE
apache-spark131 1.3.1 (new formula)

### PR DESCRIPTION
**Pull request by heuermh**
Adding new formula for apache-spark version 1.3.1 as of this commit

https://github.com/Homebrew/homebrew/blob/8dd3478131b232553f4c0cab197f8443a0a559a6/Library/Formula/apache-spark.rb

### CODE REVIEW

File: apache-spark131.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require "formula"
```
Comment (DomT4) [inline apache-spark131.rb]: Can scrap this.

File: apache-spark131.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+class ApacheSpark < Formula
+  desc "Engine for large-scale data processing"
+  homepage "https://spark.apache.org/"
+  head "https://github.com/apache/spark.git"
```
Comment (DomT4) [inline apache-spark131.rb]: `head` should be beneath the `url`, `version` and `sha`.

File: apache-spark131.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  url "https://d3kbcqa49mib13.cloudfront.net/spark-1.3.1-bin-hadoop2.6.tgz"
+  version "1.3.1"
+  revision 1
```
Comment (DomT4) [inline apache-spark131.rb]: Scrap the revision.

File: apache-spark131.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  sha1 "86911b6c8964230a93691bd45589f491c10d36c0"
```
Comment (DomT4) [inline apache-spark131.rb]: Please use `sha256`.

**Comment by DomT4** [comment]
Needs a `conflicts_with "apache-spark", :because => "Differing version of same formula"` line as well.

**Comment by heuermh** [comment]
Thank you for the review comments!

File: apache-spark131.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+class ApacheSpark < Formula
```
Comment (DomT4) [inline apache-spark131.rb]: Missed this one in my review, Sorry! Class name should match the formula name, in this case `ApacheSpark131`

File: apache-spark131.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  desc "Engine for large-scale data processing"
+  homepage "https://spark.apache.org/"
+  url "https://d3kbcqa49mib13.cloudfront.net/spark-1.3.1-bin-hadoop2.6.tgz"
+  sha256 "a25aaf58cfb3c64e3c77bdf9dde1a61247846d60519dd28b18d60d162d19c79a"
+  head "https://github.com/apache/spark.git"
+  version "1.3.1"
```
Comment (DomT4) [inline apache-spark131.rb]: Version line goes directly beneath the `url` line.

**Comment by DomT4** [comment]
Couple more nits but nearly there :)

**Comment by heuermh** [comment]
No problem, still learning on my end.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/d649fd2b8c4f3d491468acc1b1fc8fe9263f93c8. Thank you for your contribution to Versions, we appreciate it! :smiley_cat:

**Comment by heuermh** [comment]
Thank you!

**Pull request closed without merging**